src/transformers/utils/sentencepiece_model_pb2.py

```diff
@@ -2,7 +2,7 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
This file was a copy of a file in the sentencepiece library. We need it to load the sentencepiece model into a python object for the conversion of sentencepiece tokenizer into fast ones. 

This file was [updated in December 2020](https://github.com/google/sentencepiece/commit/7e87d686783bf09489dae8475f6b626a41b50273) and then in [January 2021](https://github.com/google/sentencepiece/commit/8eaa672a374cbaf2761358055dd971d49cfff0c0) as the sentencepiece model was augmented with several new features including byte fallback.

Reply:
The changes you link are within the compiled proto file, I looked at the `*.proto` files and found this huge commit:

https://github.com/google/sentencepiece/commit/329383b455a5795f3d182159eb0985a3f20f0fa2#diff-31916168ddf65a6b69364c6c819c13b2024f148d3f84f64f89bc7b3d1e8ddf74R146

Which contains more values to check against maybe:

`split_digits`
`required_chars`
`byte_fallback`
`vocabulary_output_piece_score`
`train_extremely_large_corpus`

Don't think they are all used at runtime, but maybe worth a look too (`split_digits` looks like a good candidate)